lib/ssl/src/tls_connection_1_3.erl

```diff
@@ -604,6 +604,7 @@ do_client_start(ServerHello, State0) ->
 
 initial_state(Role, Sender, Host, Port, Socket, {SSLOptions, SocketOptions, Trackers}, User,
 	      {CbModule, DataTag, CloseTag, ErrorTag, PassiveTag}) ->
+    put(log_level, maps:get(log_level, SSLOptions)),
```

Comment:
Because then we don't have to carry ssl_options as as argument to a lot of functions just to be able log
something at deep down in the call stack.

I asked about this in our meeting.